maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/LiferayMavenWorkspaceProject.java

```diff
@@ -62,7 +62,7 @@ public LiferayMavenWorkspaceProject(IProject project) {
 	}
 
 	@Override
-	public String getLiferayHome() {
+	public String getBundleHome() {
```

Comment:
Do we have to rename this from getLiferayHome to getBundleHome?  can we keep it `getLiferayHome`?

Reply:
ok, I will revert this name.